test/files/rules/nounusedvariable-imports.test.ts

```diff
@@ -10,3 +13,5 @@ $(_.xyz());
 module S {
   var template = '';
 }
+
+import * as foo from "lib";
```

Comment:
Can you do a little false positive checking here too, please? Make sure this doesn't trigger if it is actually used.